sonar-plugin-api/src/main/java/org/sonar/api/ce/measure/test/TestIssue.java

```diff
@@ -80,12 +80,13 @@ public String severity() {
 
   @Override
   @CheckForNull
+  @Deprecated
   public Duration debt() {
-    return debt;
+    return effort();
   }
 
   public Duration effort() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1161)